package.json

```diff
@@ -22,21 +26,21 @@
       "html"
     ]
   },
+  "browserify": {
+    "transform": [
+      "babelify"
+    ]
+  },
   "author": "Alexander Zonov",
   "license": "MIT",
   "dependencies": {},
   "devDependencies": {
     "babel-jest": "^18.0.0",
     "babel-preset-es2015": "^6.22.0",
     "eslint": "^3.15.0",
-    "jest": "^18.1.0",
+    "jest": "^21.2.1",
```

Comment:
All tests are checked using jest 18.
Why you change jest version?
Is version 21 full compatible with 18?